set.mm

```diff
@@ -1682,6 +1682,21 @@ giving a shorter proof but depending on more axioms (namely, ~ ax-1 and
       ( wn wi pm2.18 syl ) ABDBEBCBFG $.
   $}
 
+  $( Elimination of a nested antecedent as a partial converse of ~ ja (the
+     other partial converse being ~ jarri ).  (Contributed by Wolf Lammen,
```

Comment:
Maybe we can remove the remark abount partial converse, which is more appropriate in the comment of jarli ?

Reply:
done

Reply:
I would have removed "as a partial converse of ~ ja" since the partial converse of ~ ja is actually jarli, as indicated in the comment of jarli. But if you want to keep it, it's as you prefer too.

Reply:
also done